primestg/report/reports.py

```diff
@@ -1184,6 +1184,14 @@ def get_concentrator(self, objectified_concentrator):
         return concentrator
 
     @property
+    def supported(self):
+        if self.report_type in ('S02', 'S04', 'S05', 'S06', 'S09', 'S12',
+                                'S13', 'S15', 'S17'):
```

Comment:
I think you can move this list into a global variable inside the module. e.g. `SUPPORTED_REPORTS` and then do:
```python
    return self.report_type in SUPPORTED_REPORTS
```